Test/Import-WhiskeyPowerShellModule.Tests.ps1

```diff
@@ -0,0 +1,137 @@
+Set-StrictMode -Version 'Latest'
+
+& (Join-Path -Path $PSScriptRoot -ChildPath 'Initialize-WhiskeyTest.ps1' -Resolve)
+
+$testRoot = $null
+$output = $null
+
+function Init
+{
+    $Global:Error.Clear()
+    $script:testRoot = New-WhiskeyTestRoot
+}
+
+function GivenModuleInstalledLocally
+{
+    Mock -CommandName 'Test-Path' -ModuleName 'Whiskey' -MockWith { $true }
+    Mock -CommandName 'Import-Module' -ModuleName 'Whiskey' -MockWith { $true }
+}
+
+function GivenModuleInstalledGlobally
+{
+    Mock -CommandName 'Test-WhiskeyPowershellModule' -ModuleName 'Whiskey' -MockWith { $true }
+    Mock -CommandName 'Import-Module' -ModuleName 'Whiskey' -MockWith { $true }
+}
+
+function GivenModuleNotInstalledLocally
+{
+    Mock -CommandName 'Test-Path' -ModuleName 'Whiskey' -MockWith { $false } 
+}
+
+function GivenModuleNotInstalledGlobally
+{
+    Mock -CommandName 'Test-WhiskeyPowershellModule' -ModuleName 'Whiskey' -MockWith { $false }
+}
+
+function Import-PowerShellModule
+{
+    param(
+        [String[]]$moduleName
+    )
+
+    $parameter = @{
+        'Name' = $moduleName;
+        'PSModulesRoot' = $testRoot;
+    }
+
+    $script:output = Invoke-WhiskeyPrivateCommand -Name 'Import-WhiskeyPowerShellModule' -Parameter $parameter -ErrorAction $ErrorActionPreference
+}
+
+function Reset
+{
+    Reset-WhiskeyTestPSModule
+}
+
+function ThenImportsLocalModule
+{
+    Assert-MockCalled -CommandName 'Test-Path' -ModuleName 'Whiskey' -Times 1

```

Comment:
We don't need this assertion, just need to assert that we called `Import-Module` with the expected module name.